css-grid-1/grid-items/grid-item-containing-block-001.xht

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <title>CSS Grid Layout Test: A grid item is sized within the containing block defined by its grid area</title>
```

Comment:
This tittle sounds like a good "assert" and too long for a title.
Maybe you can find something shorter for the title.